std/memory.asm

```diff
@@ -0,0 +1,73 @@
+use std::convert::int;
+use std::utils::cross_product;
+use std::utils::unchanged_until;
+use std::array;
+
+// A read/write memory, similar to that of Polygon:
+// https://github.com/0xPolygonHermez/zkevm-proverjs/blob/main/pil/mem.pil
+machine Memory(LATCH, m_is_write) {
+
+    // lower bound degree is 65536
+
+    operation mload<0> m_addr, m_step -> m_value;
+    operation mstore<1> m_addr, m_step, m_value ->;
+
+    call_selectors selectors;
```

Comment:
what's `call_selectors`?

Reply:
They were introduced in #1126 as a way for a machine to reference its selector witness columns, which are generated dynamically by the linker.